core/src/main/scala/scalaz/Catchable.scala

```diff
@@ -16,7 +16,6 @@ package scalaz
  * total.
  */
 ////
-@deprecated("No laws, and Throwable is not referentially transparent. Prefer MonadError", "7.2.21")
```

Comment:
Was the deprecation of `Catchable` contentious?

Reply:
I assume so. I think it should be deprecated.